### DISCUSSION TITLE
enumeration in html vs revealjs

### DISCUSSION THREAD

**Discussion by hardin47** [quarto-dev/quarto-cli#1897] [Q&A]
it seems that when i change the format from `html` to `revealjs`, the enumeration changes.  in fact, with `revealjs`, I can't figure out the correct markdown to get the second layer of enumeration to be letters instead of numbers.  is there a magic indent that I'm missing?  can anyone help me get letters in the `revealjs` enumeration?

```
---
title: "clicker"
format:
  revealjs:
    footnotes-hover: true
---
<!-- the two formats are html and revealjs -->

# Clicker Q

to go with **Probability & Statistics** by DeGroot and Schervish.  Math 152 - Statistical Theory.

---

1. The Central Limit Theorem (CLT) says:[^1]
   a. The sample average (statistic) converges to the true   average (parameter)
   b. The sample average (statistic) converges to some pont
   c. The distribution of the sample average (statistic)    converges to a normal distribution
   d. The distribution of the sample average (statistic)    converges to some distribution
   e. I have no idea what the CLT says
   

[^1]:  c. The distribution of the sample average (statistic) converges to a normal distribution

---

2. Which cab company was involved?[^2]
...

```

`revealjs` rendering:

![image](https://user-images.githubusercontent.com/7210585/184690713-d295a09c-7ae5-4db7-8581-0e2420407274.png)

`html` rendering:

![image](https://user-images.githubusercontent.com/7210585/184690842-25d4d3d1-5fcb-4dfa-8e9b-511f429a47d9.png)

**Comment by mcanouil** [comment]
I think it is somehow related to the following css and something else (a lua filter or else) 

https://github.com/quarto-dev/quarto-cli/blob/52b04fecac7f91210a4781f8af89762ac9b35e18/src/resources/formats/revealjs/reveal/css/theme/template/theme.scss#L117-L119

To note, the issue is not the nested part, but that 'decimal' type is used for any ordered list no matter the type define in `<ol>` tags.

**Reply by hardin47** [reply]
i'm using a custom .scss file.  do you have a suggestion for what i can put into that file to get letter enumeration at the second level?

thanks!

**Reply by mcanouil** [reply]
Here what happens when you tweak the css from the browser:

https://user-images.githubusercontent.com/8896044/184698010-f962020d-915b-4587-8969-0ef032640824.mov


The tricky part is, you can't really override this, because it will overrides all `<ol>` elements.
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/184697889-2fea9901-7671-4297-ba6b-de288a6f362d.png">

This is why there is some post-processing involved here but don't know if it's Quarto, Pandoc or Reveal.JS 🤷‍♂️

**Comment by jjallaire** [comment]
Could you try using the selector `.reveal ol ol` ?

**Reply by hardin47** [reply]
yes, that worked!  I just put this in my .qmd file (didn't need to put it in the separate .scss file, but my guess is that it would work there, too):

````
```{=html}
<style>
.reveal ol ol {
list-style-type: lower-alpha;
}
</style>

```
````

thank you sooooo much!!

**Reply by mcanouil** [reply]
@jjallaire Why is the ordered list not resolved the same way in `revealjs` and `html` format?
It feels like this is a workaround to fix the fact that for some reason the ordered list attributes is ignored in `revealjs` (_i.e._, `type="a"` and `type="1"` in this example)

**Reply by jjallaire** [reply]
Because they have entirely different stylesheets (one is bootstrap HTML and one is RevealJS)

**Reply by mcanouil** [reply]
So, the type attributes being ignored might be an issue from Reveal.JS? (a while ago, I saw some issues regarding ordered list in Reveal.JS).
